### ISSUE TITLE
Undefined 'lib_qt4'

### ISSUE THREAD

**Issue by Nenuial** [qgis/homebrew-qgisdev#37]
Trying to compile the latest qgis3-dev formula ended with the following error:

```
Error: undefined local variable or method 'lib_qt4' for #<Formulary::FormulaNamespacee06509da8ce28e7c35872f88367a4f0d::Qgis3Dev:0x007ff194a187a8>
/usr/local/Homebrew/Library/Taps/qgis/homebrew-qgisdev/Formula/qgis3-dev.rb:316:in 'install'
/usr/local/Homebrew/Library/Homebrew/build.rb:139:in 'block (2 levels) in install'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1093:in 'block in brew'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1910:in 'block in stage'
/usr/local/Homebrew/Library/Homebrew/resource.rb:97:in 'block in unpack'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:14:in 'block in mktemp'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in 'block in run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in 'chdir'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in 'run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:13:in 'mktemp'
/usr/local/Homebrew/Library/Homebrew/resource.rb:93:in 'unpack'
/usr/local/Homebrew/Library/Homebrew/resource.rb:85:in 'stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1887:in 'stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1088:in 'brew'
/usr/local/Homebrew/Library/Homebrew/build.rb:114:in 'block in install'
/usr/local/Homebrew/Library/Homebrew/utils.rb:554:in 'with_env'
/usr/local/Homebrew/Library/Homebrew/build.rb:111:in 'install'
/usr/local/Homebrew/Library/Homebrew/build.rb:188:in '<main>'
```

**Comment by dakcarto** [comment]
Hi @Nenuial, thanks for reporting!

Typo should be fixed in 954c20f. Works here. Please verify.

**Comment by Nenuial** [comment]
Works !! Thanks a lot, first time I'm able to complete a build 😄
